patterns/2-structured/innersource-customer-interview-questions.md

```diff
@@ -0,0 +1,86 @@
+## **Title**
```

Comment:
```suggestion
## Title
```

Fixing the formatting of the title section.

This will also make this automatically created warning message disappear
>  Mandatory template sections [Required headline "Title" is missing...